controllers/metal3.io/action_result.go

```diff
@@ -85,7 +85,7 @@ func (r actionComplete) Dirty() bool {
 // deleteComplete is a result indicating that the deletion action has
 // completed, and that the resource has now been deleted.
 type deleteComplete struct {
-	actionComplete
+	actionComplete //nolint:unused
```

Comment:
I am a bit unsure about this piece of code and how is it used elsewhere. Hence I suppressed the linter.